src/electron/platform/android/appium-service-configurator.ts

```diff
@@ -82,9 +82,10 @@ export class AppiumServiceConfigurator implements AndroidServiceConfigurator {
         await this.adb.uninstallApk(servicePackageName);
     };
 
-    public setTcpForwarding = async (deviceId: string): Promise<void> => {
+    public setTcpForwarding = async (deviceId: string): Promise<number> => {
         this.adb.setDeviceId(deviceId);
         await this.adb.forwardPort(defaultAdbPortNumber, defaultAdbPortNumber);
+        return defaultAdbPortNumber;
```

Comment:
This is just returning a constant port right now, but the refactor to return a `Promise<number>` sets up the ability to have this pick a port dynamically in a later PR